cmd/api/config.go

```diff
@@ -51,6 +51,7 @@ type HTTPConfig struct {
 
 	RateLimInterval       string `default:"1s"`
 	MaxRequestPerInterval uint64 `default:"10"`
+	AllowList             string `default:""`
```

Comment:
maybe a comment with an example of a list of IP addrs would be helpful here.

Reply:
good idea 👍